keyboards/ziz0gaming19/c13x/readme.md

```diff
@@ -0,0 +1,27 @@
+# c13x
+
+![c13x](https://imgur.com/a/LhhLknI)
```

Comment:
From this link:

> C13X Handwired Keyboard

We would expect this to be relocated to the `keyboards/handwired/` folder.

Also,
```suggestion
![c13x](https://i.imgur.com/K0koJJW.jpeg)
```